### PR TITLE
improved footer

### PR DESCRIPTION
**Pull request by YadlaMani**

### CODE REVIEW

**Comment by YadlaMani** [comment]
Closes #1163 
@scraly

**Comment by scraly** [comment]
Hi, thanks for the PR

I have a remark:
I tested it and without the underline for a link, people don't know that a word is in fact a link.

**Comment by YadlaMani** [comment]
Will add the underline

**Comment by YadlaMani** [comment]
@scraly fixed it added underline again

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (sudhanshutech) [inline package-lock.json:1]: @YadlaMani I guess there should be no change related to package lock file. Will you revert it.

**Review by sudhanshutech — CHANGES_REQUESTED** [review]

**Comment by scraly** [comment]
You're right, the website needs a new and improved footer.

**Pull request closed without merging**